sparepo/build.py

```diff
@@ -212,8 +212,8 @@ def compute_hashtable_for_file(
             if len(particle_table) > 0:
                 clean_particle_ids[cell_id] = particle_table
 
-        cell_counts[part_type] = counts
-        cell_table[part_type] = clean_particle_ids
+        cell_counts[part_type.value] = counts
+        cell_table[part_type.value] = clean_particle_ids
```

Comment:
Indexes into dictionaries should still be the enums.

Reply:
If indexes into dictionaries are enums, then some of my other changes should be undo. Also, the docstring in line 175 and 179 of `build.py` should be modified.